pkg/controllers/state/cluster.go

```diff
@@ -55,6 +59,7 @@ func NewCluster(client client.Client, cp cloudprovider.CloudProvider) *Cluster {
 	c := &Cluster{
 		kubeClient:    client,
 		cloudProvider: cp,
+		nodesInUse:    cache.New(10*time.Second, 10*time.Second),
```

Comment:
Should we be sharing this 10 second constant with the emptiness controller?

Reply:
Thoughts on tracking this per pod, instead of via a cache? We could maintain `NominatedNodes map[pod]*Node` and clean this up as we learn about the pods real binding.

Reply:
Removed requeue from emptiness controller.  This is a more inexpensive way to track if we expect a node to be used in the near future and conveniently avoids adding more work that would be performed while holding the cluster state lock.